packages/flutter_tools/test/intellij/intellij_test.dart

```diff
@@ -0,0 +1,57 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_tools/src/base/file_system.dart';
+import 'package:flutter_tools/src/doctor.dart';
+import 'package:flutter_tools/src/intellij/intellij.dart';
+import 'package:test/test.dart';
+
+import '../src/context.dart';
+
+void main() {
+  group('IntelliJ', () {
+    group('plugins', () {
+      testUsingContext('found', () async {
```

Comment:
This test isn't hermetic and is subject to flakiness.  We try to ensure that all new tests are hermetic by using `MemoryFileSystem`, `FakePlatform`, and a mocked `ProcessManager` where need be (using context overrides).

Please update this test such that it is hermetic.

Thanks!

Reply:
Note that this test also fails inside Google (which is how this surfaced) because hermeticity is required.

Reply:
Filed https://github.com/flutter/flutter/issues/16339